scripts/jtl_listener.py

```diff
@@ -161,7 +160,12 @@ def add_result(self, success, _request_type, name, response_time, response_lengt
             connect,
             hostname
         ]
-        self.csv_results.append(self.field_delimiter.join(row))
+        # Safe way to generate csv row up to RFC4180
+        # https://datatracker.ietf.org/doc/html/rfc4180
+        # It encloses all fields in double quotes and escape single double-quotes chars with double double quotes.
+        # Example: " -> ""
+        csv_row_str = self.field_delimiter.join(['"' + x.replace('"', '""') + '"' for x in row]) + '\n'
```

Comment:
the `+ '\n'` is excessive here. The newline is added in `_flush_to_log` method

Reply:
Yep, you are right. Fixed.